Changes.md

```diff
@@ -1,6 +1,11 @@
 1.x.x.x
 =======
 
+Features
+--------
+
+- StandardAttributes : Added automaticInstancing plug to allow disabling instancing on selected locations.  Currently supported only by Arnold renderer.
```

Comment:
Backticks around `automaticInstancing` will improve the formatting in the documentation and on the releases page.